packages/account/src/Containers/employment-tax-info/employment-tax-info.tsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+//@ts-nocheck [TODO] - Need to fix typescript errors in Autocomplete & SelectNative components
+
+import { useMemo, useRef } from 'react';
+import { Form, Formik, FormikValues } from 'formik';
+import clsx from 'clsx';
+import { localize } from '@deriv-com/translations';
+import { useDevice } from '@deriv-com/ui';
+import EmploymentTaxDetailsContainer from '../employment-tax-details-container';
+import { getEmploymentAndTaxValidationSchema } from '../../Configs/user-profile-validation-config';
+import ScrollToFieldWithError from '../../Components/forms/scroll-to-field-with-error';
+import { Div100vhContainer, FormSubmitButton, Modal, ThemedScrollbars } from '@deriv/components';
+import { useTinValidations } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import './employment-tax-info.scss';
+
+type TEmploymentTaxInfoProps = {
+    disabled_items: string[];
+    getCurrentStep: () => number;
+    onSave: (current_step: number, values: FormikValues) => void;
+    onCancel: (current_step: number, goToPreviousStep: () => void) => void;
+    onSubmit: (
+        current_step: number,
+        values: FormikValues,
+        setSubmitting: (is_submitting: boolean) => void,
+        goToNextStep: () => void
+    ) => void;
+    goToPreviousStep: () => void;
+    goToNextStep: () => void;
+    real_account_signup_target: string;
+    value: FormikValues;
+};
+
+const EmploymentTaxInfo = observer(
+    ({
+        disabled_items,
+        value,
+        getCurrentStep,
+        onSave,
+        onCancel,
+        onSubmit,
+        goToPreviousStep,
+        goToNextStep,
+        real_account_signup_target,
+    }: TEmploymentTaxInfoProps) => {
+        const { isMobile, isDesktop } = useDevice();
+        const scroll_div_ref = useRef(null);
+        const { tin_validation_config, mutate } = useTinValidations();
+        const { client } = useStore();
+
+        const editable_fields = useMemo(
+            () =>
+                ['employment_status', 'tax_residence', 'tax_identification_number'].filter(
+                    field => !disabled_items.includes(field)
+                ) || [],
+            [disabled_items]
+        );
+
+        const is_eu = real_account_signup_target === 'maltainvest';
```

Comment:
for personal details sections, we dont have real_account_signup_target. isnt?
shouldnt we check for current_landing_company in personal details section in account settings ?

Reply:
This wrapper is used in real accocunt sign up. For personal details, the check is different